modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -10,6 +10,9 @@ import 'rxjs/add/operator/map';
 import * as models                                           from '../model/models';
 import { BASE_PATH, COLLECTION_FORMATS }                     from '../variables';
 import { Configuration }                                     from '../configuration';
+{{#withInterfaces}}
+import { I{{classname}} }                                    from './{{classname}}Interface';
```

Comment:
That's a good point. I'm okay with both options but have a slight preference for the clearer naming convention. Let's see what the others think about it.